kANe/v2p1/01_Topics_of_dharmashAstra.md

```diff
@@ -1,266 +1,177 @@
-+++
-title = "01 Topics of Dharmaśāstra"
++++ title = "01 Topics of Dharmaśāstra"
```

Comment:
Please revert this line. It is important (for html rendering) that it remain as it was earlier.